### ISSUE TITLE
yuzu installer does not work with a VPN or proxy

### ISSUE THREAD

**Issue by zhaowenlan1779** [yuzu-emu/yuzu#1573]
So that's it.. I have always believed that the installer won't work behind a proxy, but was pretty surprised when I found that it cannot work with VPN connected either. The VPN I'm using is a simple L2TP connection, and yuzu installer won't open with it connected. Instead, a "Web navigation cancelled" message is shown, and if I refresh the page, it shows the common IE error "Cannot open this page" instead. When a proxy is connected, I get the same message.

I'm not sure if this can be fixed, but I believe this is something that deserves to be considered an issue. As this installer has become official, I decided to open an issue here so that it doesn't get ignored. (actually should probably have opened it in j-selby/liftinstall instead...)

The screenshots are in Chinese, but I'll post them anyway:
![image](https://user-images.githubusercontent.com/21307832/47441520-e33ff380-d7e2-11e8-92ff-df906e589424.png)
![image](https://user-images.githubusercontent.com/21307832/47441528-e6d37a80-d7e2-11e8-8f00-c5ee8c086bbb.png)

VPN configuration (third line means "Username and password"):
![image](https://user-images.githubusercontent.com/21307832/47441709-4f225c00-d7e3-11e8-8dcb-4bfa18aa1add.png)

**Comment by EmulationChannel** [comment]
@zhaowenlan1779 
https://github.com/yuzu-emu/liftinstall/releases/download/1.2/yuzu_install.exe

**Comment by j-selby** [comment]
This is misleading. The installer works fine with VPNs and proxies. This appears to be some crazy configuration error on your end - i.e 'localhost' isn't reachable, or your OS' settings for web content.

**Comment by zhaowenlan1779** [comment]
Then how come I can use the installer fine with VPN disconnected? The VPN is directly configured in Windows settings. anyway, I will do more test though.

**Comment by j-selby** [comment]
Regardless, this doesn't belong here, and you acknowledged that in your issue. I am unable to replicate this with my own VPN tool (protonvpn), and there are no specific issues for Windows where this would be an issue other than something crazy with your own configuration.

Requesting to close.

**Comment by zhaowenlan1779** [comment]
So, it's not like localhost is unreachable or anything. I can run and use `gem server` (with the vpn connected, ofc) perfectly well.
```
PS C:\WINDOWS\system32> gem server
Server started at http://[::]:8808
Server started at http://0.0.0.0:8808
::1 - - [25/Oct/2018:22:11:20 中国标准时间] "GET / HTTP/1.1" 200 127187
- -> /
::1 - - [25/Oct/2018:22:11:20 中国标准时间] "GET /gem-server-rdoc-style.css HTTP/1.1" 200 4313
http://localhost:8808/ -> /gem-server-rdoc-style.css
```
Interestingly, `gem server` doesn't work either behind proxy (it returns a HTTP 502), but it **does** work with the same VPN connected.

> there are no specific issues for Windows where this would be an issue other than something crazy with your own configuration.

I don't think this is a good attitude towards issues. It's not like I have encountered any other problems with my current configuration, and I don't think I have tweaked the network config at all.

The windows hosts file only contains a single line (related) which I think was there by default.
```
127.0.0.1 localhost
```

> I am unable to replicate this with my own VPN tool (protonvpn)

I don't think this is a reliable point. The fact that it works with a specific VPN doesn't necessarily mean that it will work with other types of protocols and software.

I'll close this for now (as this is in the wrong repo anyway), but will return to provide more information after I tested this on a clean Windows VM.
